common/utils/src/main/scala/org/apache/spark/internal/LogKey.scala

```diff
@@ -16,399 +16,399 @@
+  case object RELATION_NAME extends LogKey
+  case object RELATIVE_TOLERANCE extends LogKey
+  case object REMAINING_PARTITIONS extends LogKey
+  case object REPORT_DETAILS extends LogKey
+  case object RESOURCE extends LogKey
+  case object RESOURCE_NAME extends LogKey
+  case object RESOURCE_PROFILE_ID extends LogKey
+  case object RESOURCE_PROFILE_IDS extends LogKey
+  case object RETRY_COUNT extends LogKey
+  case object RETRY_INTERVAL extends LogKey
+  case object RIGHT_EXPR extends LogKey
+  case object RMSE extends LogKey
+  case object RPC_ENDPOINT_REF extends LogKey
+  case object RULE_BATCH_NAME extends LogKey
+  case object RULE_NAME extends LogKey
+  case object RULE_NUMBER_OF_RUNS extends LogKey
+  case object RUN_ID extends LogKey
+  case object SCHEMA extends LogKey
+  case object SCHEMA2 extends LogKey
+  case object SERVICE_NAME extends LogKey
+  case object SESSION_HOLD_INFO extends LogKey
+  case object SESSION_ID extends LogKey
+  case object SESSION_KEY extends LogKey
+  case object SHARD_ID extends LogKey
+  case object SHUFFLE_BLOCK_INFO extends LogKey
+  case object SHUFFLE_ID extends LogKey
+  case object SHUFFLE_MERGE_ID extends LogKey
+  case object SHUFFLE_SERVICE_NAME extends LogKey
+  case object SIZE extends LogKey
+  case object SLEEP_TIME extends LogKey
+  case object SLIDE_DURATION extends LogKey
+  case object SMALLEST_CLUSTER_INDEX extends LogKey
+  case object SPARK_DATA_STREAM extends LogKey
+  case object SPARK_PLAN_ID extends LogKey
+  case object SQL_TEXT extends LogKey
+  case object SRC_PATH extends LogKey
+  case object STAGE_ATTEMPT extends LogKey
+  case object STAGE_ID extends LogKey
+  case object START_INDEX extends LogKey
+  case object STATEMENT_ID extends LogKey
+  case object STATE_STORE_PROVIDER extends LogKey
+  case object STATUS extends LogKey
+  case object STDERR extends LogKey
+  case object STORAGE_LEVEL extends LogKey
+  case object STORAGE_LEVEL_DESERIALIZED extends LogKey
+  case object STORAGE_LEVEL_REPLICATION extends LogKey
+  case object STORE_ID extends LogKey
+  case object STREAM_ID extends LogKey
+  case object STREAM_NAME extends LogKey
+  case object SUBMISSION_ID extends LogKey
+  case object SUBSAMPLING_RATE extends LogKey
+  case object SUB_QUERY extends LogKey
+  case object TABLE_NAME extends LogKey
+  case object TABLE_TYPES extends LogKey
+  case object TARGET_PATH extends LogKey
+  case object TASK_ATTEMPT_ID extends LogKey
+  case object TASK_ID extends LogKey
+  case object TASK_NAME extends LogKey
+  case object TASK_SET_NAME extends LogKey
+  case object TASK_STATE extends LogKey
+  case object TEMP_FILE extends LogKey
+  case object TEMP_PATH extends LogKey
+  case object TEST_SIZE extends LogKey
+  case object THREAD extends LogKey
+  case object THREAD_NAME extends LogKey
+  case object TID extends LogKey
+  case object TIME extends LogKey
+  case object TIMEOUT extends LogKey
+  case object TIMER extends LogKey
+  case object TIMER_LABEL extends LogKey
```

Comment:
`TIMER_LABEL` and `TIME_UNITS` are not being used, should we temporarily clean them up?

Reply:
Let me remove it.
When all pr for the structured logs is completed, I will also submit a PR to `improve` the names of this class, such as `NUM-FILES` and `WEIGHTED-NUM`. Should `NUM` be used as the prefix or suffix? We need to unify. @gengliangwang @LuciferYang

Reply:
prefix sounds good